src/styles/blocks/header.scss

```diff
@@ -0,0 +1,62 @@
+.header {
+  padding: 0 $header-paddind-inline;
+  box-shadow: 0 2px 4px 0 $header-color-shadow;
+
+  @include flex-container(flex, row, space-between, center);
+
+  &_logo {
+    text-decoration: none;
+    display: flex;
+
+    &_img {
+      margin: 10px auto;
+
+      @include logo-image-size(40px);
+    }
+  }
+}
+
+.nav {
+  display: flex;
+
+  &__list {
+    display: flex;
+    list-style: none;
+
+    @include reset-margin-padding(0, 0);
+  }
+
+  &__item {
+    display: flex;
+    margin: 0 0 0 20px;
```

Comment:
The margin shorthand property is used here. It's a good practice to specify all 4 sides (top, right, bottom, left) for better readability. In this case, it should be margin: 0 0 0 20px;.